_posts/builtin/2022-02-23-container-orchestration-provider.md

```diff
@@ -14,52 +14,66 @@ The Container Orchestration service is composed of two bundles, one that exposes
+
+- **Image Download Timeout** - Describes the amount of time the framework will let the image download before timeout.
   
-- **Internal Ports** - This field accepts a comma separated list of ports that will be internally exposed on the spun up docker container.
+- **Internal Ports** - This field accepts a comma-separated list of ports that will be internally exposed on the spun-up container.
   
-- **External Ports** - This field accepts a comma separated list of ports that will be externally exposed on the host machine.
+- **External Ports** - This field accepts a comma-separated list of ports that will be externally exposed on the host machine.
   
 - **Privileged Mode** - This flag if enabled will give the container root capabilities to all devices on the host system. Please be aware that setting this flag can be dangerous, and must only be used in exceptional situations.
   
-- **Environment Variables (optional)** - This field accepts a comma separated list of environment variables, which will be set inside the container when spun up.
+- **Environment Variables (optional)** - This field accepts a comma-separated list of environment variables, which will be set inside the container when spun up.
   
-- **Volume Mount (optional)** - This field accepts a comma separated list of system-to-container file mounts. This allows for the container to access files on the host machine.
+- **Volume Mount (optional)** - This field accepts a comma-separated list of system-to-container file mounts. This allows for the container to access files on the host machine.
   
-- **Peripheral Device (optional)** - This field accepts a comma separated list of device paths. This parameter allows for devices to be passed through from the host to the container.
+- **Peripheral Device (optional)** - This field accepts a comma-separated list of device paths. This parameter allows for devices to be passed through from the host to the container.
 
-- **Logger Type** - This field provides a dropdown selection of supported container logging drivers.
-  
-- **Logger Parameters (optional)** - This field accepts a comma separated list of logging parameters. More information can be found in the [Docker logger documentation](https://docs.docker.com/config/containers/logging/configure/#supported-logging-drivers).
-  
+- **Logger Type** - This field provides a drop-down selection of supported container logging drivers.
+
+- **Logger Parameters (optional)** - This field accepts a comma-separated list of logging parameters.
+
+After specifying container parameters, ensure to set **Enabled** to **true** and press **Apply**. The container engine will then pull the respective image, spin up and start the container. If the gateway is reset (power cycled), and the container and Container Orchestration Service are set to **enabled**, The framework will automatically start the container again upon startup. 
```

Comment:
> If the gateway is reset (power cycled), and the container and Container Orchestration Service are set to **enabled**, The framework will automatically start the container again upon startup.

If the gateway or the framework are power cycled, and the container and Container Orchestration Service are set to **enabled**, the framework will automatically start the container again upon startup.